documentation/src/docs/controller-service.md

```diff
@@ -154,296 +151,298 @@ Controller Process Diagram
 
 #### Stream State
- Znode which captures the state of the stream. It is an enumerator with
+ Znode which captures the state of the Stream. It is an enumerator with
  values from *creating, active, updating, scaling, truncating, sealing,*
- and *sealed*. Once _active_, a stream transitions between performing a
- specific operation and active until it is sealed. A transition map is
+ and *sealed*. Once _active_, a Stream transitions between performing a
+ specific operation and remanis _active_ until it is sealed. A transition map is
  defined in the
- [State](https://github.com/pravega/pravega/blob/master/Controller/src/main/java/io/pravega/controller/store/stream/tables/State.java)
+ [State](https://github.com/pravega/pravega/blob/master/Controller/src/main/java/io/pravega/controller/store/Stream/tables/State.java)
  class which allows and prohibits various state transitions.
- Stream state describes the current state of the stream. It transitions
+ Stream State describes the _current state_ of the Stream. It transitions
  from _active_ to respective action based on the action being performed
- on the stream. For example, during scaling the state of the stream
+ on the Stream. For example, during scaling the state of the Stream
  transitions from *active* to *scaling* and once scaling completes, it
- transitions back to *active*. Stream state is used as a barrier to
- ensure only one type of operation is being performed on a given stream
+ transitions back to *active*. Stream State is used as a barrier to
+ ensure only one type of operation is being performed on a given Stream
  at any point in time. Only certain state transitions are allowed and
- are described in the State Transition object. Only legitimate state
+ are described in the state transition object. Only legitimate state
  transitions are allowed and any attempt for disallowed transition
  results in appropriate exception.
 
 #### Truncation Record
- This corresponds to the stream cut which was last used to truncate the
- given stream. All stream segment queries superimpose the truncation
- record and return segments that are strictly greater than or equal to
- the stream cut in truncation record.
+ This corresponds to the `StreamCut` which was last used to truncate the
+ given Stream. All Stream Segment queries superimpose the truncation
+ record and return Stream Segments that are strictly greater than or equal to
+ the `StreamCut` in truncation record.
 
 #### Sealed Segments Record
- Since the segment table is append only, any additional information
- that we need to persist when a segment is sealed is stored in sealed
- segments record. Presently, it simple contains a map of segment number
+ Since the Segment Table is append only, any additional information
+ that we need to persist when a Stream Segment is sealed and is stored in sealed
+ Stream Segments record. At present, it contains a map of Stream Segment number
```

Comment:
@shiveshr  any updates on  this ` At present, it contains a map of Stream Segment number`

Reply:
All of this needs to change to incorporate new metadata schema.
We will need to rewrite this entire section.
I will take a stab at it sometime next week if its ok.

Reply:
content provided